cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -175,7 +179,11 @@ if (NOT THRIFT_FOUND)
   set(THRIFT_HOME "${THRIFT_PREFIX}")
   set(THRIFT_INCLUDE_DIR "${THRIFT_PREFIX}/include")
   IF (${UPPERCASE_BUILD_TYPE} STREQUAL "DEBUG")
-    set(THRIFT_STATIC_LIB "${THRIFT_PREFIX}/lib/libthriftd.a")
+    IF (MSVC)
+      set(THRIFT_STATIC_LIB "${THRIFT_PREFIX}/lib/thriftmdd.lib")
```

Comment:
as @wesm mentioned, in case of thrift the required lib is with MD suffix, but others 3rd party dependencies of Parquet might be named without MD suffix at all. So it seems additional checks should be placed somewhere in any case, why not to use this solution at least for now :)